README.textile

```diff
@@ -1,5 +1,20 @@
 h1. Elasticsearch
 
+h2. What to Use Elasticsearch For and When to Use It.
+
+Have extremely large amounts of data to churn through? Want to return searches quickly and accurately? Elasticsearch can easily index massive amounts of data and return results faster with less resource consumption when compared to other search technologies (e.g. conventional SQL database search).
```

Comment:
This line is okay, but I'd prefer to remove any mention of other systems, perhaps:

"Have extremely large amounts of data to churn through? Want to return searches quickly and accurately? Elasticsearch can easily index massive amounts of data and return results extremely fast."